### ISSUE TITLE
logging-too-many-args erroneously reported when logging byte string

### ISSUE THREAD

**Issue by PhilippSelenium** [PyCQA/pylint#2938]
### Steps to reproduce
test_logging.py
```
import logging
log = logging.getLogger()
log.debug(b"foo%s", b"bar")
```

Run
```
pylint -E test_logging.py
```

### Current behavior
test_logging.py:5:0: E1205: Too many arguments for logging format string (logging-too-many-args)

### Expected behavior
No error is reported since pipeing the example into ipython works just fine.

### pylint --version output
pylint 2.3.1
astroid 2.2.5
Python 3.6.7 (default, Oct 22 2018, 11:32:17) 
[GCC 8.2.0]

**Comment by msjyoo** [comment]
@PCManticore Similar issue happens when logging dictionaries. https://github.com/python/cpython/blob/master/Lib/logging/__init__.py#L310 Should I create a new issue?

**Comment by PCManticore** [comment]
@msjyoo Yes, please!
